src/BugReporting.jl

```diff
@@ -17,7 +17,22 @@ function check_rr_available()
     end
 end
 
-function normalize_inner_trace(trace_directory)
+function collect_inner_traces(trace_directory)
+    # If this is already a trace directory, return just it
+    if isfile(joinpath(trace_directory, "version"))
+        return [trace_directory]
+    end
+    # If this is a directory containing traces, return those
+    traces = String[]
+    for f in readdir(trace_directory, join=true)
```

Comment:
Skip `latest-trace` to avoid attempting to pack that twice?

Reply:
I've done one better; it will now automatically skip stuff that has `mmap_pack_*` files inside of it.